### DISCUSSION TITLE
What else add to Nylo?

### DISCUSSION THREAD

**Discussion by andim27** [nylo-core/nylo#155] [Ideas]
Please add into core(if it's possible) next: ( isInternetConnected() or getInternetStatus()->false||true)

**Comment by agordn52** [comment]
Hi @andim27,

Hope you're well.
Yeah, I could squeeze this into the next release.

Maybe one method on the `Nylo` instance and another in Base `ApiService` 👍
We could even have a new Widget too. I'll experiment.
